README.md

```diff
@@ -439,11 +428,29 @@ references. I.e. `pc.getTranscievers()` returns objects that can be
 retained and owned by the caller. This pattern is fine for garbage
 collected or reference counted languages, but not great with Rust.
 
-```
-Dec 18 11:33:06.850  INFO str0m: MediaData(MediaData { mid: Mid(0), pt: Pt(104), time: MediaTime(3099135646, 90000), len: 1464 })
-Dec 18 11:33:06.867  INFO str0m: MediaData(MediaData { mid: Mid(0), pt: Pt(104), time: MediaTime(3099138706, 90000), len: 1093 })
-Dec 18 11:33:06.907  INFO str0m: MediaData(MediaData { mid: Mid(0), pt: Pt(104), time: MediaTime(3099141676, 90000), len: 1202 })
-```
+### Panics, Errors and unwraps
+
+Rust adheres to [fail-last][ff]. That means rather than brushing state
+bugs under the carpet, it panics. We make a distinction between errors and
+bugs.
+
+* Errors are as a result of incorrect or impossible to understand user input.
+* Bugs are broken internal invariants (assumptions).
+
+If you scan the str0m code you find a few `unwrap()` (or `expect()`). These
+will (should) always be accompanied by a code comment that explains why the
+unwrap is okay. This is an internal invariant, a state assumption that
+str0m is responsible for maintaining.
+
+We do not believe it's correct to change every `unwrap()`/`expect()` into
+`unwrap_or_else()`, `if let Some(x) = x { ... }` etc, because doing so
+brushes an actual problem (an incorrect assumption) under the carpet. Trying
+to hobble along with an incorrect state would at best result in broken
+behavior, at worst a security risk!
+
+Panics are our friends: *panic means bug*
+
+And also: str0m should *never* panic on any user input.
```

Comment:
Maybe mention catching the panic and converting it to log error here. If the user discards the whole `Rtc` instances all the state that caused the invariant violation should be gone. This gives folks some resilience to avoid bringing down whole server processes in the event of panics

Reply:
ok